pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3322,3 +3327,11 @@ func (c *Controller) handleMemoryHotplugRequest(vm *virtv1.VirtualMachine, vmi *
 
 	return nil
 }
+
+func (c *Controller) handleDeclarativeVolumeHotplug(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) error {
+	if c.clusterConfig.HotplugVolumesEnabled() || !c.clusterConfig.DeclarativeHotplugVolumesEnabled() {
```

Comment:
Would it make sense logging this? Since it ignores declarative hotplug because of the old hotplug?